File: source/server/types/Page.go
package types

type Page struct {
	PageNum    int64         `json:"pageNum"`
	PageSize   int64         `json:"pageSize"`
	TotalPage  int64         `json:"totalPage"`
	TotalCount int64         `json:"totalCount"`
	TotalMoney float64       `json:"totalMoney"`
	PageData   []interface{} `json:"pageData"`
}

File: ui/src/model/page.ts
export interface Page<T> {
  pageNum: number
  pageSize: number
  totalPage: number
  totalCount: number
  totalOut: number
  totalIn: number
  notInOut: number
  pageData: T[]
}

export interface Result<T> {
  c: number
  m: string
  d: T
}
